gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun Sep 16 17:38:31 CDT 2018
+#Wed Jun 05 09:47:38 MDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
Should we run `./gradlew wrapper --gradle-version 5.1.1` to update the `.sh` and `.bat` scripts as well?

We could also add the following to the `./build.gradle` file:

```
wrapper {
    distributionType = Wrapper.DistributionType.ALL
}
```

https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
I updated the scripts using the `./gradlew wrapper --gradle-version 5.1.1` command in 071d952.